Musicplayer.php

```diff
@@ -13,28 +13,37 @@
 	</head>
 <body>
 	<div class ="grid-container">
-		<div class="img_1">
-			<img src = "images/img.png">
-		</div>
 		<div class="navigation">
+			<div class="logo">
+				<a href = "homepage.php"><img src = "images/logo8.png" height = "100"></a>
+			</div>
 			<nav>
 				<ul>
-					<li><a href = "index.html">Home</a></li>
-					<li><a href = "index.html">Music Play</a></li>
-					<li><a href = "index.html">Contact</a></li>
-					<li><a href = "01_login_real.php">Account</a></li>
+					<li><a href = "homepage.php">Home</a></li>
+					<li><a href = "musicplayer.php">Music Play</a>
+						<ul>
+							<li><a href= "Query1.php">Query1</a></li>
+							<li><a href= "Query2.php">Query2</a></li>
+						</ul>
+					</li>
+					<li><a href = "login.php">Contact</a></li>
+					<li><a href = "login.php">LoginOut</a></li>
+					<li><a href = "signup.php">SignUp</a></li>
 				</ul>
 			</nav>
 		</div>
-		<div class="content_1"><!-- Holds the main page content -->
-			<div class="section3"><!-- Holds the main page connect -->
-				<h1>All the song upload in this playlist.Total duration of playlist:</h1>
-			</div>
+		
+		<!-- Div that holds the banner -->
+		<div class="img_1">
+			<img src = "images/img.png">
 		</div>
 		<div class="content_2">
-			<h1>PlayList</h1>
+			<img src = "images/img.png">
```

Comment:
21/9: Unlike layout one, Layout two doesn't have a side bar so content_2 div box in layout two wasn't a side bar. It is only a div box to hold the img. So I also remove the text that say play list and I also I release that this h1 was I h1 I delete above when it set the text-align center.